### PR TITLE
Initial commit for Documents

### PR DESCRIPTION
**Pull request by Arkinn**
First release of the documentation with all requested changes completed.

### CODE REVIEW

File: guide/index.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+ - [Review flagged content](moderator/reviewflag.md)
+ - [Edit someones post](moderator/editpost.md)
+ - [Delete someones post](moderator/deletepost.md)
+ - [Rename a discussion](moderator/renamediscussion.md)
+ - [Lock a discussion](moderator/lock.md)
+ - [Sticky/Unsticky a discussion](moderator/sticky.md)
+ - [Edit assigned tags](moderator/editassignedtags.md)
+
+### Section 3: Admin Basics
```
Comment (jordanjay29) [inline guide/index.md]: Here's where you could make a general "logging into the Administrator Panel" and link to the others that use it. As I mentioned in a comment above, the repetition might not be useful for our readers. Others might want to weigh in here (Toby, Franz, Sijad) for what the aim is for these doc pages.

File: guide/index.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+ - [Change the Forum Title](admin/forumtitle.md)
+ - [Change the Forum Description](admin/forumdescription.md)
+ - [Change home page view](admin/homeview.md)
+ - [Edit the Welcome Banner](admin/welcomebanner.md)
+ - [Add/Modify/Delete User Groups](admin/usergroups.md)
+ - [Assign permissions to User Groups](admin/grouppermissions.md)
+ - [Upload a custom Logo / Favicon](admin/uploadlogo.md)
+ - [Create/Edit/Remove Primary & Secondary Tags](admin/tags.md)
+ 
+ ### Section 4: Useful Links
```
Comment (jordanjay29) [inline guide/index.md]: I'd actually put the Glossary (and I'd name it Glossary) up above Registration under #1.

File: guide/like.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+## User Handbook
+### Liking a post
```
Comment (jordanjay29) [inline guide/like.md]: Might want to note somewhere here that it requires the Likes extension to be enabled.

File: guide/login.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
+## User Handbook
+### Logging In
+
+Once you have [created a user account](https://github.com/Arkinn/docs/blob/testing/guide/registration.md) the next step to interacting with the community will be to login.
+
+The login screen can be accessed by clicking **Log In** at the top right of the screen, the animation below will show you what this looks like:
+
+![Animation - Login page](687474703a2f2f692e696d6775722e636f6d2f616c56484e53702e676966.gif)
+
+Enter your **Username** _OR_ **e-mail address** and **Password** that you filled out during the [sign up process](https://github.com/Arkinn/docs/blob/testing/guide/registration.md) and then click Log In.
```
Comment (jordanjay29) [inline guide/login.md]: Some communities will have SSO extensions (Facebook, Twitter, Github, and others) configured, and if the user registered through those they will **have** to log in via those buttons (unless they [migrate to a regular login method](https://discuss.flarum.org/d/5744-how-to-migrate-an-o-auth-account-to-regular-account)). This should be part of the documentation because those users won't have passwords to enter.

File: guide/moderator/deletepost.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+
+As always, firstly you'll want to find the post that needs deleting. Once you have, hover your mouse to the right, lower side of the post area  to bring up the _post menu_ click on that and then click **Delete**.
+
+Be aware though, clicking **Delete** will only soft-delete the post, meaning for now, the post will remain there in non-live state. This gives you the opportunity to restore the post should you think you have made a mistake, or want another mod to review it, etc.
+
+The option to Delete looks like this:
+
+![Animation - Showing post soft-delete](687474703a2f2f692e696d6775722e636f6d2f52666c6c5351782e676966.gif)
+
+To permanently delete the post, once you are **sure you want it removing** then go ahead and expand the post, by clicking on the _expand button_ next to the username of the poster, once the post is expanded, do the same as you did in the last step, click on the _post menu_, this will bring up the option to **Delete Forever** as shown below:
```
Comment (jordanjay29) [inline guide/moderator/deletepost.md]: "..sure you want it **remov*ing***..."

Grammar? Shouldn't it be *removed*?

File: guide/moderator/editassignedtags.md [hunk 6]
```diff
@@ -0,0 +1,15 @@
+## User Handbook
+
+### Editing tags Assigned to a Thread
+
+Has a thread been assigned the wrong tags or do you need to add more? not to worry! this is easy to do.
```
Comment (jordanjay29) [inline guide/moderator/editassignedtags.md]: Sorry, really trying not to nitpick on prose, but sentences start with capital letters. "**N**ot to worry!" **T**his is easy to do."

File: guide/moderator/lock.md [hunk 7]
```diff
@@ -0,0 +1,11 @@
+## User Handbook
+
+## Locking a Discussion
+
+To lock a discussion, as always you'll first need to find the discussion in threaded view, once you have go ahead and bring up the thread menu by hovering the mouse over the right of the thread, and selecting the three dots menu, as show in the screenshot below:
```
Comment (jordanjay29) [inline guide/moderator/lock.md]: "threaded view"

This is the (All) Discussions List. Threaded View, in forum lingo, often means something else relating to type of post display mode (where posts nest under the post they reply to, rather than the linear form we use).

File: guide/moderator/lock.md [hunk 8]
```diff
@@ -0,0 +1,11 @@
...
+
+![SS - Showing thread menu](687474703a2f2f692e696d6775722e636f6d2f4558665258415a2e706e67.png)
+
+With the menu, click on the **Lock** label which will immediately lock the thread, preventing any futher replies. A confirmation of the thread lock will be a padlock over the user's avatar as shown below:
```
Comment (jordanjay29) [inline guide/moderator/lock.md]: It will allow replies from users who can lock discussions. Not sure if you need to mention this or not.

File: guide/moderator/reviewflag.md [hunk 9]
```diff
@@ -0,0 +1,39 @@
...
+ - Delete the Post
+
+Is the content truly requiring mod action? then go ahead and delete the post! just be warned that this option will only **soft-delete** the post for now (we'll show you how to permanently delete it below shortly).
+![Animation - showing post soft-deletion](687474703a2f2f692e696d6775722e636f6d2f52666c6c5351782e676966.gif)
+
+ - Dismiss the Flag
+
+Don't agree that this content needs any action? then just go ahead and **Dismiss the Flag** this will then _cancel_ the report, and act as if nothing ever happened. (literally!)
+
+### Deleting the Content
```
Comment (jordanjay29) [inline guide/moderator/reviewflag.md]: I really like this write-up, but you already have a doc on deleting posts. Is this necessary? 

(Then again, I want to keep it just because it's so well done.)

File: guide/moderator/sticky.md [hunk 10]
```diff
@@ -0,0 +1,16 @@
+## User Handbook
+
+### Sticky/Unsticky a thread
```
Comment (jordanjay29) [inline guide/moderator/sticky.md]: Let's note here what Sticky does (and that the extension must be enabled to have this feature). Sticky posts rise to the top of the All Discussions list until a user has read all new posts within. They will always live at the top of their respective tag's discussion list.

File: guide/profile.md [hunk 11]
```diff
@@ -0,0 +1,46 @@
+## User Handbook
+### Your Profile
+
+Flarum has a basic profile system, in your profile there are 3 different categories of your profile that you can view.
+
+ - Posts
+ - Discussions
+ - Mentions
+
+## Categories Explained
```
Comment (jordanjay29) [inline guide/profile.md]: I wouldn't call them categories. Maybe sections? 

Small nitpick here. Dismiss if you want.

File: guide/profile.md [hunk 12]
```diff
@@ -0,0 +1,46 @@
...
+You can be taken straight to any of the posts, or discussions explained in this section by clicking on the result of each of the categories.
+
+### Posts 
+The Posts tab will allow you to see a 'Timeline view' of posts that you have made in any discussion you have access to, the most recent posts you have made will appear at the top, whilst the older posts will be descending.
+You can see an example of how the look in the screenshot below:
+
+![SS - Posts category](687474703a2f2f692e696d6775722e636f6d2f42756538425a312e706e67.png)
+
+### Discussions
+The Discussions tab will allow you to see what discussions you are currently participating in, or have started. The order they appear in is the same as **Posts**, the most recent at the top.
```
Comment (jordanjay29) [inline guide/profile.md]: Discussions will only show those you have started.

File: guide/profile.md [hunk 13]
```diff
@@ -0,0 +1,46 @@
...
+### Mentions
+The Mentions tab will display all the posts you have recieved a mention in, following the same ordering by having the most recent at the top.
+
+This is how it looks:
+
+![SS - Mentions category](687474703a2f2f692e696d6775722e636f6d2f34536762687a4c2e706e67.png)
+
+### Settings
+
+The Settings page allows you to configure certain preferences on your profile, you may change your registered e-mail address and password on this page. You can also set your preference on how you wish to be notified about certain events in Flarum, for example you can set if you wish to be notified by email and/or in the notification area if someone replies to your post. There are many other options available for you to see, take a look at the screenshot below to see what Flarum can do for you!
```
Comment (jordanjay29) [inline guide/profile.md]: Might note that this is only for the user themselves, no one else can see this. Could also note the above categories (sections) are public to anyone, but will only show posts/discussions that they have access to.

File: guide/registration.md [hunk 14]
```diff
@@ -0,0 +1,17 @@
+## User Handbook
+### Registration
+
+To be able to post, and interact with a Flarum powered community, you'll first need to register yourself a user account. The registration process could not be easier, and takes only a few moments of your time.
+
+To sign up, click the "Sign Up" button at the top right of the screen to be presented with the sign up form, like the animation below:
```
Comment (jordanjay29) [inline guide/registration.md]: Like on the Login page, some communities will have SSO options for Facebook, Github, Twitter, etc, as an alternative that would be good to note here. Maybe even screenshot.

File: guide/startdiscussion.md [hunk 15]
```diff
@@ -0,0 +1,30 @@
...
+
+Select the tags you think are appropriate, or relevant to your discussion, and then click "Ok" to apply them.
+
+### Discussion Title
+
+Discussion Title is giving a very short, brief subject on your discussion, for example if you wanted to start a discussion about the weather today, then your Title would be something along the lines of "The Weather Today".
+
+### Write a post
+
+This is where you expand in more detail about what you want to talk about, feel free to go crazy here! (although, not too crazy!).
```
Comment (jordanjay29) [inline guide/startdiscussion.md]: (although, not too crazy!)

Like they should follow the rules of their community? ;)

File: guide/admin/forumtitle.md [hunk 16]
```diff
@@ -0,0 +1,23 @@
...
+![Forum Title picture](687474703a2f2f692e696d6775722e636f6d2f58514f57504e712e706e67.png)
+
+Replace the text with your desired forum title and then click on _**Save Changes**_
+
+Once you've one that, to the top left of your screen, there will be an arrow pointing to the left as show below:
+
+![Anim for Home](687474703a2f2f692e696d6775722e636f6d2f35514a593466482e676966.gif)
+
+Click the arrow, which will then take you back to your forum's main page, with your new forum title in place.
+a
```
Comment (Arkinn) [inline guide/admin/forumtitle.md]: Whoops, typo. Sorted.

File: guide/index.md (hunk 1 above)
Comment (Arkinn) [inline guide/index.md]: My note on the previous issue for this explains my reasoning, interested to know what others think though.

File: guide/moderator/deletepost.md (hunk 5 above)
Comment (Arkinn) [inline guide/moderator/deletepost.md]: Yes! it should be, thanks for pointing that out. Sorted.

File: guide/moderator/lock.md (hunk 7 above)
Comment (Arkinn) [inline guide/moderator/lock.md]: Ok I got you, I'll fix that shortly.

File: guide/moderator/lock.md (hunk 8 above)
Comment (Arkinn) [inline guide/moderator/lock.md]: I think I will add it, as it's better to have full clarification than unanswered questions for the future. Will do this shortly.

File: guide/moderator/sticky.md (hunk 10 above)
Comment (Arkinn) [inline guide/moderator/sticky.md]: Good point, I'll add that to the doc shortly.

File: guide/profile.md (hunk 11 above)
Comment (Arkinn) [inline guide/profile.md]: Hmm, I'll have a think on this one.

File: guide/profile.md (hunk 12 above)
Comment (Arkinn) [inline guide/profile.md]: Ah, again wasn't aware of that, I'll change it shortly.

File: guide/startdiscussion.md (hunk 15 above)
Comment (Arkinn) [inline guide/startdiscussion.md]: Haha, believe me that statement covers a whole broad range of things! :P

File: guide/admin/forumdescription.md [hunk 17]
```diff
@@ -0,0 +1,21 @@
...
+
+Once you are in the Administration Panel, you'll see some navigation tabs flowing down the left hand side, click on **Basics** to reveal the configuration page to the right hand side.
+
+![Basics Screenshot](687474703a2f2f692e696d6775722e636f6d2f5a4d6767745a472e706e67.png)
+
+On the configuration page to the right, you'll see **Forum Description** with a box underneath to write in, write a few words which generally describe what kind of forum community you are creating then click on **Save Changes**.
+
+Once you've one that, to the top left of your screen, there will be an arrow pointing to the left as show below:
+
+![Arrow to Home](687474703a2f2f692e696d6775722e636f6d2f35514a593466482e676966.gif)
```
Comment (Arkinn) [inline guide/admin/forumdescription.md]: Just to clarify on this, you'd like to see a written description **instead** of the gif?

File: guide/admin/usergroups.md [hunk 18]
```diff
@@ -0,0 +1,53 @@
...
+ - Singular: is how the name is written should it only apply to one person, in this example our Singular name is **Staff Member**
+ - Plural: is how the name is written should it apply to more than one person, this example would be named **Staff Members**
+ 
+#### Color
+
+Choosing a Color can be done by obtaining the HTML Colour Code of your desired colour, there are plenty of websites you can view which a quick Google search would give you. Enter the colour code you wish to use (starts with a **#**).
+
+#### Icon
+
+The Icon is a small picture that goes at the side of your user avatar in posts to signify that you are a member of this particular user group. To pick your icon, you'll need to visit [this website](http://fontawesome.io/icons/) to pick out your icon, for this example we will be using the ```id-card``` icon. In this field, you'll just need to write "id-card" (without the quotation brackets) and then we're done!
```
Comment (Arkinn) [inline guide/admin/usergroups.md]: Agreed, I'll fix that shortly.

File: guide/admin/forumdescription.md (hunk 17 above)
Comment (jordanjay29) [inline guide/admin/forumdescription.md]: No, I mean the description in the gif says false, it might make it easier to follow if the gift showed entering a sample description.

File: guide/admin/forumdescription.md [hunk 19]
```diff
@@ -0,0 +1,21 @@
+## User Handbook
+### Changing the Forum Description
+
+
+Changing the description of your forum instance requires you to be in the Administrator Control Panel, you can do this by clicking your username to the top right of your screen, and then clicking **Administration**. The example below will show you how this is done:
```
Comment (Arkinn) [inline guide/admin/forumdescription.md]: I thought long and hard about this, when we link to these documents in the forums, mostly it will be single pages. I included that as a standard at the top of the pages because A) itś in sequence with what needs to be done (I.E. They need to be in the Admin CP) and B) It prevents threads in the forum on *How do I get to that panel?* I did think about putting a notice in the index, but then we would in turn have to link to that before passing them the thread (and hope they read it). By having it at the top of the pages, itś much more difficult to miss. That said, I am happy to re-think it if its really necessary.

File: guide/admin/forumdescription.md (hunk 17 above)
Comment (Arkinn) [inline guide/admin/forumdescription.md]: I am with you, agreed yes it would be better I will add it during edits.

File: guide/admin/usergroups.md [hunk 20]
```diff
@@ -0,0 +1,53 @@
...
+### Create Group Form Components
+
+#### Name
+
+ - Singular: is how the name is written should it only apply to one person, in this example our Singular name is **Staff Member**
+ - Plural: is how the name is written should it apply to more than one person, this example would be named **Staff Members**
+ 
+#### Color
+
+Choosing a Color can be done by obtaining the HTML Colour Code of your desired colour, there are plenty of websites you can view which a quick Google search would give you. Enter the colour code you wish to use (starts with a **#**).
```
Comment (Arkinn) [inline guide/admin/usergroups.md]: Ok that can happen, that is just the brit in me coming out, we spell that differently over here. I will change all mentions of it to be Color on edit.

File: guide/admin/forumdescription.md (hunk 19 above)
Comment (jordanjay29) [inline guide/admin/forumdescription.md]: Makes sense, agreed. Consider this one resolved.

File: guide/admin/usergroups.md (hunk 20 above)
Comment (Arkinn) [inline guide/admin/usergroups.md]: Fixed this instance.

File: guide/admin/usergroups.md (hunk 18 above)
Comment (Arkinn) [inline guide/admin/usergroups.md]: This is fixed.

File: guide/admin/usergroups.md [hunk 21]
```diff
@@ -0,0 +1,53 @@
...
+
+Once you've filled in the form, it should look something like this:
+
+![SS Group Create Filled In](687474703a2f2f692e696d6775722e636f6d2f336863613430662e706e67.png)
+
+Click on **Save Changes** once you've completed the form, which will then create your new user group!
+
+### Delete a User Group
+
+Deleting a user group is very simple, at the top of the configuration page, you'll see all the user groups that you've created, click on the user group that you wish to **delete** which will bring up the user group editing box, bring your mouse to the bottom right of that box over the **Delete group** link, click that and then click **OK** to the warning that comes up. Boom! the user group is now deleted.
```
Comment (Arkinn) [inline guide/admin/usergroups.md]: I have re-written the section, including a screenshot to aid visually. Can you let me know what you think of the new section? in testing repo.

File: guide/admin/homeview.md [hunk 22]
```diff
@@ -0,0 +1,33 @@
+## User Handbook
+### Changing the Home Page View
```
Comment (Arkinn) [inline guide/admin/homeview.md]: Maybe something like *Switching Home Page display options*? does that sound better you think? I would like to try and keep it at Home Page because that is what it is listed as in the ACP, for sake of consistency.

File: guide/admin/homeview.md [hunk 23]
```diff
@@ -0,0 +1,33 @@
...
+
+Changing the home page view layout of your forum instance requires you to be in the Administrator Control Panel, you can do this by clicking your username to the top right of your screen, and then clicking **Administration**. The example below will show you how this is done:
+
+![Administration Link Screenshot](687474703a2f2f692e696d6775722e636f6d2f575a315a6a78392e676966.gif)
+
+Once you are in the Administration Panel, you'll see some navigation tabs flowing down the left hand side, click on **Basics** to reveal the configuration page to the right hand side.
+
+![Basics Screenshot](687474703a2f2f692e696d6775722e636f6d2f5a4d6767745a472e706e67.png)
+
+On the configuration page, you'll see the **Home Page** option with 2 radio buttons underneath, they read **All Discussions** and **Tags**.
```
Comment (Arkinn) [inline guide/admin/homeview.md]: This one is sorted, removed the location and added 2 separate notes.

File: guide/admin/homeview.md [hunk 24]
```diff
@@ -0,0 +1,33 @@
...
+
+### All Discussions
+
+Having this radio button set will display the most recent discussions by default on the forum main page (home page), sticky topics will of course remain at the very top of the board.
+
+The **All Discussions** option makes the forum look like this:
+
+![Screenshot of All Discussions](687474703a2f2f692e696d6775722e636f6d2f5256417a6d57662e706e67.png)
+
+### Tags
```
Comment (Arkinn) [inline guide/admin/homeview.md]: As above, sorted (finished them both the same time).

File: guide/admin/tags.md [hunk 25]
```diff
@@ -0,0 +1,57 @@
+## User Handbook
+### Creating, editing and deleting Tags
```
Comment (Arkinn) [inline guide/admin/tags.md]: Added notice (copied from previous), so this one is sorted.

File: guide/admin/tags.md [hunk 26]
```diff
@@ -0,0 +1,57 @@
...
+
+The slug is a string that goes into the URL, essentially it's a version of the name of the tag that is all lower-case, and uses dashes instead of spaces.
+
+Unless you need to, you generally don't need to do anything with this, as when you type in the **Name** the slug auto-fills in.
+
+#### Description:
+
+The description of the tag will appear alongside the **name** in the tag selection post when making a post, it's a short few words giving a more in-depth description of the category it belongs to.
+
+#### Colour:
```
Comment (Arkinn) [inline guide/admin/tags.md]: Changed all references from Colour -> Color

File: guide/admin/homeview.md (hunk 22 above)
Comment (jordanjay29) [inline guide/admin/homeview.md]: I'm trying to figure out how to keep both the consistent terminology and try to pick up on key words that are regularly used in support requests. Many are looking for a category view or tags only, but I'm not sure how to word that without overdoing it. 

Maybe we just use what you suggested and see what happens.

Comment (Luceos) [inline guide/admin/homeview.md]: If you consider this from a search perspective, adding "view" here has no value. The bottom line is the page allows for configuring the home page, no matter the technical implementation.

tldr drop "View" 😁

File: guide/admin/tags.md [hunk 27]
```diff
@@ -0,0 +1,57 @@
...
+### Deleting a Tag
+
+To delete a tag, hover your mouse to the right of the tag you wish to remove, and click on the **Pencil Icon** this will bring up the editing prompt, to the bottom right of that prompt, you'll see the **Delete tag** link, click that and then click **OK** to delete the tag.
+The below animation will show you how this is done:
+
+![Anim - Delete a Tag](687474703a2f2f692e696d6775722e636f6d2f635664654a7a772e676966.gif)
+
+### Editing a Tag
+
+Made a mistake, or need to change something on a tag? follow the same procedure as above in **Deleting a Tag** until you get to the editing prompt, this time, **DON'T** click on Delete Tag, instead edit the information displayed until you are happy, then click **Save Changes**. Easy as that!
```
Comment (Arkinn) [inline guide/admin/tags.md]: Re-wrote the section, merged them together (with some extra screenshots), and added part about hide from all discussions. Fixed.

File: guide/admin/welcomebanner.md [hunk 28]
```diff
@@ -0,0 +1,21 @@
+## User Handbook
+### Editing the Welcome Banner
+
+The Welcome Banner is one of the first thing you forum guests, members, moderators and admins will see. From a fresh installation, it is the text situated inside the large grey strip at the top of the page.
```
Comment (Arkinn) [inline guide/admin/welcomebanner.md]: I've added a note underneath explaining that, sorted.

File: guide/credits.md [hunk 29]
```diff
@@ -0,0 +1,13 @@
+## User Handbook
+### Credits
+
+There are a couple of people on the Flarum team that special thanks are due to for their help, and sharing their expertise to make this user guide.
+
+- [Webeindustry](https://discuss.flarum.org/u/webeindustry)
```
Comment (Arkinn) [inline guide/credits.md]: Added my own credit :)

File: guide/flag.md [hunk 30]
```diff
@@ -0,0 +1,17 @@
+## User Handbook
+### Flagging user content
+
+Every now and then content that you think is breaking the rules needs to be looked at by a Moderator/Admin. This is super-easy to do with Flarum so let's get started.
+
+Firstly, find the post that you wish to flag, then click on the _Post Menu_ to reveal the **flag** button. Click that and the box below will display.
+
+We need to let the Moderators know why this post is being flagged, you can select from a list of 3 items, these are currently:
```
Comment (Arkinn) [inline guide/flag.md]: Sorted this one!

File: guide/index.md (hunk 2 above)
Comment (Arkinn) [inline guide/index.md]: I've sorted that, added it above section 1.

File: guide/like.md (hunk 3 above)
Comment (Arkinn) [inline guide/like.md]: Noted at the start, before any instructions. Sorted.

File: guide/login.md (hunk 4 above)
Comment (Arkinn) [inline guide/login.md]: I've added a small section detailing that, sorted!

File: guide/moderator/editassignedtags.md (hunk 6 above)
Comment (Arkinn) [inline guide/moderator/editassignedtags.md]: No sorry required, I cannot learn if people don't point these things out - sorted.

File: guide/admin/tags.md [hunk 31]
```diff
@@ -0,0 +1,67 @@
...
+
+![SS - Tag Editor](tag-editor.png)
+
+To make changes, modify the content in the desired text boxes and then click **Save Changes**.
+
+***Hide from All Discussions*** is an option used to prevent any Discussions this tag is attached to from being displayed on the **All Discussions** home page view.
+
+The **Delete Tag** button also resides at the bottom-right hand of the Tag Editor, to remove the tag, follow the animation below:
+
+![Anim - Delete a Tag](687474703a2f2f692e696d6775722e636f6d2f635664654a7a772e676966.gif)
```
Comment (jordanjay29) [inline guide/admin/tags.md]: Not sure if this GIF is necessary. It's pretty self-explanatory, you click the button and it deletes the tag.

File: guide/moderator/lock.md (hunk 7 above)
Comment (Arkinn) [inline guide/moderator/lock.md]: This is fixed.

File: guide/moderator/lock.md (hunk 8 above)
Comment (Arkinn) [inline guide/moderator/lock.md]: Fixed this one.

File: guide/moderator/reviewflag.md (hunk 9 above)
Comment (Arkinn) [inline guide/moderator/reviewflag.md]: As such it's not completely necessary, but I'd rather keep it in as it's part of the review process, and makes the write up more complete (from start to finish). I'll consider this one completed unless you really feel it needs changing.

File: guide/moderator/sticky.md (hunk 10 above)
Comment (Arkinn) [inline guide/moderator/sticky.md]: This one is fixed, added the note about making sure the extension is enabled, and explained what the sticky is.

File: guide/profile.md (hunk 11 above)
Comment (Arkinn) [inline guide/profile.md]: Yes, I agree with this one, I've changed the label from category to section on all of them.

File: guide/profile.md (hunk 12 above)
Comment (Arkinn) [inline guide/profile.md]: Fixed, removed the part about participating in.

File: guide/profile.md (hunk 13 above)
Comment (Arkinn) [inline guide/profile.md]: Added both those notes. Sorted.

File: guide/registration.md (hunk 14 above)
Comment (Arkinn) [inline guide/registration.md]: Completed, explained about SSO options on registration, and added a screenshot on the registration page.

File: guide/admin/forumdescription.md (hunk 17 above)
Comment (Arkinn) [inline guide/admin/forumdescription.md]: I edited the section to make it look better, added some screenshots instead of gifs as I felt it was better suited that way.

File: guide/admin/homeview.md (hunk 22 above)
Comment (Arkinn) [inline guide/admin/homeview.md]: Changed the title on page, and index page. Considering completed.

File: guide/admin/tags.md [hunk 32]
```diff
@@ -0,0 +1,57 @@
...
+
+This is the colour you would like the tag to appear in on the posts, and forums, the colour box uses **HTML Colour Codes** beginning with a **#** you can Google around for HTML Colour Codes, there are plenty of websites on this.
+
+Once you've filled in the appropriate information in the Create Tag prompt, click on **Save Changes** to add the new tag.
+
+### Moving the Tag between Primary, and Secondary tags
+
+By default, new tags created will be **Secondary Tags** also known as **Sub Categories**, sometimes you'll want these to be primary tags, to do this, click and drag the tag to the **Primary** slot at the top, follow the below animation to see how this is done:
+
+![Anim - Moving Tags](687474703a2f2f692e696d6775722e636f6d2f765738796e42732e676966.gif)
```
Comment (Arkinn) [inline guide/admin/tags.md]: I added it to the creating tag section, and added a screenshot showing the result.

File: guide/admin/uploadlogo.md [hunk 33]
```diff
@@ -0,0 +1,20 @@
...
+
+To upload a custom logo to your forum instance requires you to be in the Administrator Control Panel, you can do this by clicking your username to the top right of your screen, and then clicking **Administration**. The example below will show you how this is done:
+
+![Administration Link Screenshot](687474703a2f2f692e696d6775722e636f6d2f575a315a6a78392e676966.gif)
+
+Once you are in the Administration Panel, you'll see some navigation tabs flowing down the left hand side, click on **Appearance** to reveal the configuration page to the right hand side.
+
+![Appearance Screenshot](687474703a2f2f692e696d6775722e636f6d2f736a6a6b455a362e706e67.png)
+
+Under the **Logo** label, you'll find a button which is labelled **Choose an image...**, click that which will bring up the file upload prompt, find the image you wish to upload them click **Open**
```
Comment (jordanjay29) [inline guide/admin/uploadlogo.md]: Dismissing this one. Considering this may change per server, it's too much to try to explain to users. Favicon sizes are well recorded online, and users can experiment with logo sizes or just ask on Flarum's discuss.

**Comment by Luceos** [comment]
I'll give merging a go right now @jordanjay29

**Comment by Luceos** [comment]
Okay merged.
